param/reactive.py

```diff
@@ -755,14 +787,56 @@ def __init__(
         ]
         self._setup_invalidations(depth)
         self._kwargs = kwargs
-        self.rx = reactive_ops(self)
+        self._rx = reactive_ops(self)
```

Comment:
This enables me to provide a docstring. Don't know how to do that without it.

Reply:
`reactive_ops` (reactive operators) and `rx` (reactive expressions) are two different classes and should not have the same docstring. I actually did not understand there was a difference before I started to inspect the code and add docstrings.

Reply:
I added pytests for the 3 docstrings that should be identical. Thx.